sdk/monitor/Microsoft.Azure.Monitor.OpenTelemetry.Exporter/src/AzureMonitorConverter.cs

```diff
@@ -71,18 +75,22 @@ internal static TelemetryItem GeneratePartAEnvelope(Activity activity)
             return telemetryItem;
         }
 
-        internal static void ExtractRoleInfo(Resource resource, out string roleName, out string roleInstance)
```

Comment:
nit: consider renaming this to something like `InitRoleInfo`.
In my opinion, that's more representative of what this method is doing. 
But totally fine to leave it as it is. Your call. :)

Reply:
I was looking out for a good name for this method. `InitRoleInfo` looks good to me.